packages/beacon-node/src/chain/archiver/index.ts

```diff
@@ -301,21 +303,37 @@ export class Archiver {
       finalizedCanonicalCheckpointsCount,
       finalizedFoundCheckpointsInStateCache,
     });
-    this.logger.info("Attached validators finalized proposal stats", {
-      ...attachedValidators,
-      finalizedAttachedValidatorsCount,
-    });
 
-    this.metrics?.allValidators.expected.set(allValidators.expected);
+    // Only log to info if there is some relevant data to show
+    //  - No need to explicitly track SYNCED state since no validators attached would be there to show
+    //  - debug log if validators attached but no proposals were scheduled
+    //  - info log if proposals were scheduled (canonical) or there were orphans (non canonical)
+    if (finalizedAttachedValidatorsCount !== 0) {
+      if (attachedValidators.total !== 0 || attachedValidators.orphaned !== 0) {
+        this.logger.info("Attached validators finalized proposal stats", {
+          ...attachedValidators,
+          finalizedAttachedValidatorsCount,
```

Comment:
Not a fan of `finalizedAttachedValidatorsCount` as it is way to hard to read

```suggestion
          validators: finalizedAttachedValidatorsCount,
```

We do not omit any information by simplifying this as
- we know that those are attached validators as it is already part of message
- what are even finalized validators? This name might make sense in code but not in the log
- should be obvious it is the count as we show a number `validators=10`